app/src/main/java/net/wetfish/wetfish/ui/settings/MySettingsActivity.java

```diff
@@ -0,0 +1,227 @@
+package net.wetfish.wetfish.ui.settings;
```

Comment:
Once again, more comments would be ideal.

Reply:
I'll look it over and see if I can put some m ore comments on an upcoming commit. Thanks!